src/main.rs

```diff
@@ -93,6 +94,13 @@ fn print_file<P: AsRef<Path>>(
     line_changes: &Option<LineChanges>,
 ) -> Result<()> {
     let mut highlighter = HighlightFile::new(filename.as_ref(), syntax_set, theme)?;
+    if let Some(lang) = options.language {
+        let syntax = syntax_set.syntaxes().iter()
+            .find(|s| s.name.to_lowercase() == lang.to_lowercase());
+        if syntax.is_some() {
```

Comment:
Can we use `if let Some(syntax) = syntax {...}` here to avoid the `.unwrap()`?

Reply:
Hmm, not sure why I didn't do it that way 😕